### DISCUSSION TITLE
How can I change the name from "Portfolio"  into "Research Project" ?

### DISCUSSION THREAD

**Discussion by yuenwanho** [mmistakes/minimal-mistakes#2662] [Q&A]
Hi,

I want to do some simple modifications to your existing template, but I am unsure how I can make it and wish to seek your comments. I like your "Portfolio" and I am looking for solutions on how I can change the name from "Portfolio" into "Research", such that the directory and path of portfolio files will be: "Home / Research/ Foo Bar Identity" Instead of "Home / Portfolio / Foo Bar Identity". I tried to modify 

```
collections:
  research:
    output: true
    permalink: /:collection/:path/
```

and update the portfolio.md

```
---
title: Portfolio
layout: collection
permalink: /research/
collection: research
entries_layout: grid
classes: wide
---

```

as well as include

```
main: 
- title: "Research"
    url: /research/

``` 
in the navigation.yml. 

However, the modification still shows "Home / Portfolio / Foo Bar Identity". when I click the Foo Bar Identity in the portfolio page. 

Thanks for helping!

**Comment by mmistakes** [comment]
Hard to say without seeing your site repo, but the breadcrumb trail links come from the page URL. Take a look at the Foo Bar Identity page, what's it's URL?

https://github.com/mmistakes/minimal-mistakes/blob/master/_includes/breadcrumbs.html
